docker/end-user-ui/Dockerfile

```diff
@@ -15,4 +15,4 @@ WORKDIR /tmp/end-user-ui/dist
 
 EXPOSE 8888
 
-CMD ["http-server", "-p", "8888"]
+CMD ["http-server", "-p", "8080"]
```

Comment:
8080 is fine with me... but what about the IG welcome page currently on localhost:8080?

Here's what I get after the `skaffold dev` step in <https://github.com/ForgeRock/forgeops-init/tree/master/6.5/oauth2/development>:

```
$ lsof -i :8080
COMMAND  PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
kubectl 9822 mark    6u  IPv6 125886      0t0  TCP ip6-localhost:http-alt (LISTEN)
kubectl 9822 mark    7u  IPv4 125887      0t0  TCP localhost:http-alt (LISTEN)
$ curl http://localhost:8080
...
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>IG 6.5.0 Welcome Page</title>
<link rel="shortcut icon" href="http://forgerock.com/favicon.ico" />
...
<p>
Welcome to Identity Gateway 6.5.0 (IG).
</p>

<p>
Now that you have successfully installed IG,
read the <a target="_blank" href="https://backstage.forgerock.com/docs/ig/6.5">documentation</a>
to learn how to configure IG for your use.
</p>

<p><span class="copyright">Copyright 2014-2018 ForgeRock AS.</span></p>
</body>
</html>
```
I don't see any value in exposing that page, and <http://localhost:8080/openig/studio> returns 404 at present.

My feeling is that we should only expose IG's UI if we can use it to edit routes, and that IG should have its own FQDN if we do expose it.

Reply:
The reason it's exposed at all is likely due to skaffold's port-forwarding. Given that isn't something we expect people to run in production, it shouldn't really be an issue.